src/mallocTracer.cpp

```diff
@@ -91,9 +137,14 @@ void MallocTracer::initialize() {
             return strcmp(s, "malloc_hook") == 0
                 || strcmp(s, "calloc_hook") == 0
                 || strcmp(s, "realloc_hook") == 0
-                || strcmp(s, "free_hook") == 0;
+                || strcmp(s, "free_hook") == 0
+                || strcmp(s, "posix_memalign_hook") == 0
+                || strcmp(s, "aligned_alloc_hook") == 0;
         },
         MARK_ASYNC_PROFILER);
+
+    // _CS_GNU_LIBC_VERSION is not defined on musl
+    is_musl = confstr(_CS_GNU_LIBC_VERSION, NULL, 0) == 0 && errno != 0;
```

Comment:
To avoid code duplication, can we move this logic to os_linux.cpp and expose `OS::isMusl()`?